components/podIndicator/styles.js

```diff
@@ -21,36 +21,94 @@
 
 import { createStyles } from "@solid/lit-prism-patterns";
 
-const styles = (theme) =>
-  createStyles(theme, ["button"], {
+const styles = (theme, indicatorWidth, indicatorLabelWidth) => {
+  const indicatorLabelPaddingLeft = indicatorLabelWidth < 42.8 ? 45 : 40;
+  return createStyles(theme, ["button"], {
     indicator: {
       display: "flex",
+      backgroundColor: theme.palette.background.default,
+      minHeight: "60px",
+      maxHeight: "max-content",
+      width: "100vw",
+      [theme.breakpoints.up("sm")]: {
+        maxWidth: "50vw",
+        minWidth: "128px",
+        width: "max-content",
+        height: "60px",
+      },
     },
     indicatorLabel: {
-      fontSize: "0.825rem",
-      fontWeight: theme.typography.fontWeightMedium,
+      fontSize: "13px",
```

Comment:
I think we should use 0.825rem here, to stay consistent with other uses of small fonts.